skbeam/core/fitting/xrf_model.py

```diff
@@ -420,13 +420,19 @@ def add_param(self, kind, element, constraint=None):
                 if constraint is not None:
                     self._element_strategy[param_name] = constraint
                 self.params.update({param_name: new_pos})
-        else:
+        elif 'user' in element.lower():  # for user peak
```

Comment:
Can you add a note about this in the docstring for this function?  It is not immediately evident from the current docstring that `add_param` supports the notion of a `user_peak`